source/frontend/src/features/projects/list/ProjectListView.scss

```diff
@@ -0,0 +1,4 @@
+@import '~assets/scss/variables';
+@import '~assets/scss/colors';
+
+$contentheading-height: 6rem;
```

Comment:
Not sure this file is needed given most of the styles we use are directly on the page itself

Reply:
Yes, also in general we use styled components not scss for styling.